backend/substrapp/tasks/tasks.py

```diff
@@ -797,11 +811,21 @@ def prepare_volumes(subtuple_directory, tuple_type, compute_plan_key, compute_pl
 
     # local volume for train like tuples in compute plan
     if compute_plan_key is not None:
-        volume_id = get_local_folder_name(compute_plan_key)
-        get_or_create_local_volume(volume_id)
+        local_path = path.join(subtuple_directory, 'local')
```

Comment:
What about extracting this into a function `get_subtuple_local_folder`. 

The function `get_local_folder` could be renamed to `get_cp_local_folder`